### PR TITLE
add yandex searchProvider

### PR DESCRIPTION
**Pull request by 1vank1n**
Yandex geocoder has limit 25000 requests per day (https://tech.yandex.com/maps/geocoder/)

### CODE REVIEW

**Comment by caioariede** [comment]
Hey @1vank1n, thanks for your contribution! I'm currently working on support for Python 3 and Django 2.x. I'm planning to review your branch as soon as possible. In the meantime, could you please rebase your branch with master? It is failing because Travis config was outdated, and it has been fixed now.

**Comment by 1vank1n** [comment]
Hi @caioariede! Of course, I can do it. Just done.

**Comment by caioariede** [comment]
@1vank1n I think something is wrong with the rebase, look the diff

File: location_field/static/location_field/js/form.js [hunk 1]
```diff
@@ -69,17 +69,44 @@
...
-                        map.panTo(latLng);
-                    }
-                });
+                            marker.setLatLng(latLng);
+                            map.panTo(latLng);
+                        }
+                    });
+                }
+
+                if (this.options.searchProvider === 'yandex') {
```
Comment (caioariede) [inline location_field/static/location_field/js/form.js]: Can we use a `else if` here? So it's clear it's a mutually exclusive condition

**Comment by 1vank1n** [comment]
@caioariede, I have tried again. Look at this now, please.

File: location_field/static/location_field/js/form.js [hunk 2]
```diff
@@ -16,7 +16,7 @@
                     google: {
                         api: '//maps.google.com/maps/api/js',
                         mapType: 'ROADMAP'
-                    }
+                    },
```
Comment (caioariede) [inline location_field/static/location_field/js/form.js]: noice :)

File: location_field/static/location_field/js/form.js [hunk 3]
```diff
@@ -158,6 +185,10 @@
                     });
                 },
 
+                yandexSearchProvider: function (options, onload) {
+                    onload();
```
Comment (caioariede) [inline location_field/static/location_field/js/form.js]: Can we make sure it's possible to pass at least some **options**? To name a few that **I think** would be useful for someone: `apiKey`, `lang`, `kind` (not 100% sure about this one)?

**Comment by caioariede** [comment]
@1vank1n thanks man! just reviewed. it looks good, I just suggested some **nice to have™** things

**Comment by caioariede** [comment]
@1vank1n are you willing to complete this PR? You don't need to complete it by yourself, we can see if someone else wants to jump in and help you (including me). You did a great work and the only thing missing is just some polishing. What do you think?

**Comment by caioariede** [comment]
Thanks @1vank1n - can you create another PR and add yourself to AUTHORS file?

**Pull request merged**